server.js

```diff
@@ -4,9 +4,63 @@ var bodyParser = require('body-parser');
 var morgan = require('morgan');
 var config = require('./config');
 var jwt = require('jsonwebtoken');
+var Pool = require('pg').Pool;
+var bodyParser = require('body-parser');
+
+app.use(bodyParser.json());
+app.use(bodyParser.urlencoded({ extended: false }));
+app.use(morgan('dev'));
+
+var config = {
+  host: 'localhost',
+  user: 'rebeccaking',
+  password: 'sock8T0em!',
+  database: 'dareity'
+};
+var pool = new Pool(config);
+
+process.on('unhandledRejection', function(e) {
+  console.log(e.message, e.stack)
+})
 
 app.get('*', function(req, res){
-  res.json({server: 'up'})
+	pool.query('SELECT * FROM dareity_user', function(err, result){
+    if(err){
+			console.error("error",err.message);
+		} else {
+			res.json(JSON.stringify(result))
+		}
+  })
+})
+app.post('/api/create_user', function(req, res){
+  var username = req.body.name;
```

Comment:
use const instead of var
const {username, user_id,  hashed_password, is_npo} = req.body
should not be hashed_password